packages/contracts-bedrock/contracts/test/L1StandardBridge.t.sol

```diff
@@ -38,6 +38,8 @@ contract L1StandardBridge_Receive_Test is Bridge_Initializer {
     function test_receive_succeeds() external {
         assertEq(address(op).balance, 0);
 
+        vm.expectEmit(true, true, true, true);
+        emit ETHDepositInitiated(alice, alice, 100, hex"");
```

Comment:
Could we also be ok with emitting `ETHDepositInitiated` on L2? Then we could emit the event inside of `_initiateBridgeETH` which would reduce diff 

Deposit and withdrawal is all relative so i think it makes sense

cc @maurelian @smartcontracts @clabby

Reply:
It solves the issue easily and minimizes risk so I'm for it. 

All three `ETH` and all three `ERC20` events should go together then I believe.

Reply:
I think we need to verify that this won't bork indexing first. Who would be best to ask about that? (we can take this offline)

Reply:
I don't think this will impact indexing at all, fwiw there is test coverage in the indexer, integration tests

Reply:
> It solves the issue easily and minimizes risk so I'm for it.
> 
> All three `ETH` and all three `ERC20` events should go together then I believe.
> 
> Does it also remove the need for the assembly block?

Yes it removes the need for the assembly block and keeps the code much more simple. It should not break the indexer. Let me know if you have any more questions @refcell :D